mithril-common/src/entities/http_server_error.rs

```diff
@@ -45,7 +45,10 @@ pub struct ClientError {
 
 impl ClientError {
     /// ClientError factory
-    pub fn new(label: String, message: String) -> ClientError {
-        ClientError { label, message }
+    pub fn new<T1: Into<String>, T2: Into<String>>(label: T1, message: T2) -> ClientError {
```

Comment:
This would be more readable:
```suggestion
    pub fn new<L: Into<String>, M: Into<String>>(label: L, message: M) -> ClientError {
```